src/IssueViz.Security/HotspotsList/TableDataSource/HotspotTableEntry.cs

```diff
@@ -0,0 +1,128 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Windows;
+using Microsoft.VisualStudio.Shell.TableControl;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+using ImageMoniker = Microsoft.VisualStudio.Imaging.Interop.ImageMoniker;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource
+{
+    internal class HotspotTableEntry : IWpfTableEntry
+    {
+        private readonly IAnalysisIssueVisualization hotspot;
+
+        public HotspotTableEntry(IAnalysisIssueVisualization hotspot)
+        {
+            this.hotspot = hotspot;
+        }
+
+        public object Identity => hotspot;
+
+        public bool TryGetValue(string keyName, out object content)
+        {
+            var originalIssue = hotspot.Issue;
+
+            switch (keyName)
+            {
+                case StandardTableColumnDefinitions.ErrorCode:
+                    content = originalIssue.RuleKey;
+                    break;
+
+                case StandardTableColumnDefinitions.DocumentName:
+                    content = originalIssue.FilePath;
+                    break;
+
+                case StandardTableColumnDefinitions.Text:
+                    content = originalIssue.Message;
+                    break;
+
+                case StandardTableColumnDefinitions.Line:
+                    content = originalIssue.StartLine;
```

Comment:
What's the plan for showing the "live" position as the location is updated by the tagger?

Reply:
Good question. I wasn't sure if we would even want to show line/column information, as it's not exactly like the error list (e.g. we also don't show project name). If we take the data from the span, then what should we do if the span is invalid? In the error list we return "null" in GetValue which stops the issue from being rendered, but here we would probably want to keep the item in the list.
Do you think it's worth showing line/column in the first place?

Reply:
It's possible that there could be multiple hotspots of the same type in the same file. The user will need something to help disambiguate between them.

Reply:
Ok I'll handle that in a separate PR then, along with Severity column.